### ISSUE TITLE
[BUG]Can not get boot-time profile

### ISSUE THREAD

**Issue by lxsmall0624** [android/ndk#1792]
### Description

After running 
`adb root`
`adb shell setprop persist.simpleperf.boot_record 'simpleperf boot-record --enable "-a -g --duration 10 --exclude-perf"'`
`adb reboot`

I can not get the profile data under /data/simpleperf_boot_data.
Look likes simpleperf didn't work in booting.

### Affected versions

r25

### Host OS

Windows

### Host OS version

WIN10

### Affected ABIs

armeabi-v7a

### Build system

ndk-build

### minSdkVersion

extras-master-simpleperf

**Comment by yabinc** [comment]
On which android version are you using the command? The function needs support of init script in https://cs.android.com/android/platform/superproject/+/master:system/extras/simpleperf/simpleperf.rc, which doesn't exist before android T.
To help debug, can you upload result of `getprop persist.simpleperf.boot_record` and `logcat | grep simpleperf` before and after reboot?

**Comment by deadgittt** [comment]
Hello there! 
I am facing the same issue, simpleperf is not profiling booting stage.
I am running the command ```adb shell simpleperf boot-record --enable -a``` and I'm getting the following output: ```simpleperf I cmd_boot_record.cpp:83] After boot, boot profile will be stored in /data/simpleperf_boot_data```
> On which android version are you using the command? >> 15 (VanillaIceCream)

> `getprop persist.simpleperf.boot_record` >> [persist.simpleperf.boot_record]: [-a]

> `logcat | grep simpleperf >> 
01-01 00:00:14.293     1     1 I init    : Parsing file /system/etc/init/simpleperf.rc...

It is the only one log about simpleperf in android logs.

**Comment by pirama-arumuga-nainar** [comment]
yabinc may correct me when he's back from OOO.  But from what I can see, the instructions for early boot profile changed for the Android V release.  https://android.googlesource.com/platform/system/extras/+/refs/heads/main/simpleperf/doc/android_platform_profiling.md#Get-boot_time-profile has the new instructions (changed in r.android.com/2877321).

**Comment by deadgittt** [comment]
Yeah, booting stage profiling can be enabled via kernel cmdline. Thank you!
Solved
